CONTRIBUTING.rst

```diff
@@ -42,9 +42,9 @@ You can run the system tests with ``nox``::
 To run a single session, specify it with ``nox -s``::
```

Comment:
Thank you for cleaning up the text in this file. 🙏

Reply:
Of course 😀